### PR TITLE
Fix HttpAccessLogTestCase test case

### PR DESCRIPTION
**Pull request by Drifftr**
## Purpose
Fix HttpAccessLogTestCase test case
Issue - https://github.com/wso2/micro-integrator/issues/895

### CODE REVIEW

File: integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/passthru/transport/httpaccesslogs/access-log.properties [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (malakaganga) [inline integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/passthru/transport/httpaccesslogs/access-log.properties]: Shall we update the year?

Comment (Drifftr) [reply integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/passthru/transport/httpaccesslogs/access-log.properties]: Thanks. Updated

Comment (malakaganga) [reply integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/passthru/transport/httpaccesslogs/access-log.properties]: Great but we are living in 2019 shall we up another year :)

Comment (Drifftr) [reply integration/mediation-tests/tests-patches/src/test/resources/artifacts/ESB/passthru/transport/httpaccesslogs/access-log.properties]: Oops will change that :D

**Pull request merged**